### DISCUSSION TITLE
Accessing existing bindings of a Control

### DISCUSSION THREAD

**Discussion by IXimonI** [AvaloniaUI/Avalonia#16193] [Q&A]
What i'm trying to do:

- Find out the name of the Property `Abc` on the DataContext object which is bound to the attached property `Foo`.

An easy solution for this would be to retrieve the existing Avalonia.Data.Binding object. But I dont see a way to access it.
The only thing I found is that I can access the datatype of the Property `Abc` by calling `Foo.GetType()`. (Maybe I'm missing an option her to retrieve the PropertyName?)

So my question is: Is there a way to access existing bindings of an attached Property?
In WPF this can be done by using:

`BindingOperations.GetBinding(this, FooProperty);`

Thank you for your help! :)

**Comment by maxkatz6** [comment]
No, it's not currently possible to get binding instance.

**Reply by timunie** [reply]
but there is a PR #16214
